app/Http/routes.php

```diff
@@ -118,3 +118,9 @@
 	'middleware' => 'admin',
 	'uses' => 'SprintsController@delete'
 ]);
+
+Route::post('sprints', [
```

Comment:
An index on `POST /sprints` doesn't make much sense. `GET /sprints` is still free to become an index. I agree though that `/sprints/connect` is probably better as `POST /sprints` would be for sprint creation when following REST conventions.